src/fsm.c

```diff
@@ -4704,7 +4713,7 @@ ni_ifworker_do_common_bind(ni_fsm_t *fsm, ni_ifworker_t *w, ni_fsm_transition_t
 		 *   <arguments>
 		 *     <foobar type="...">
 		 *       <meta:mapping
-		 *	   	document-node="/some/xpath/expression" 
+		 *	   	document-node="/some/xpath/expression"
 		 *		skip-unless-present="true"
 		 *		/>
```

Comment:
Please remove the incompletely fixed (by some git whilespace autofix feature) chunk:
```
»·······»······· *       <meta:mapping
»·······»······· *»·····   »····document-node="/some/xpath/expression"
»·······»······· *»·····»·······skip-unless-present="true"
»·······»······· *»·····»·······/>
```
which is  covered by https://github.com/openSUSE/wicked/pull/997 (or rebase) and conflicts.

Reply:
Sorry, I pushed a old version somehow :-/